src/main/java/edu/ksu/canvas/oauth/OauthTokenRefresher.java

```diff
@@ -60,7 +60,7 @@ public TokenRefreshResponse getNewToken(String refreshToken) throws IOException
         }
         HttpEntity entity = httpResponse.getEntity();
         String responseBody = EntityUtils.toString(entity);
-        Gson gson = GsonResponseParser.getDefaultGsonParser();
+        Gson gson = GsonResponseParser.getDefaultGsonParser(false);
```

Comment:
Normally I would overload a method with a default matching the old value when adding a parameter that is most commonly still the old hardcoded value. But I could see not doing that if we want to force people to be aware of the change, and making them choose from here on out. It really depends what we're trying to achieve.

Reply:
Well I do kind of want to force `CanvasWriter` instances to pass in this value. Otherwise someone could implement a new write call that doesn't work correctly with the `serializeNulls` option. However it is unfortunate that this also impacts all the `CanvasReader` objects for which the `serializeNulls` option does not matter. And of course the method is named get**Default**GsonParser even though now we are specifying a parameter that can modify the default behavior. Naming is hard. The good thing is that this method is only used internally to the library so users won't be impacted either way.